examples/aqlm_example.py

```diff
@@ -0,0 +1,20 @@
+from vllm import LLM, SamplingParams
+
```

Comment:
I figured this was good to include, for me, as well as others.

Reply:
That sounds great, I'll do that.  People shouldn't have to modify the file.  Also, I've never written a python script with arguments either, so good practice.